templates/analytics/remote_realm_details.html

```diff
@@ -1,6 +1,9 @@
 <div class="remote-realm-information">
     <span class="label">remote realm</span>
     <h3>{{ remote_realm.name }}</h3>
+    {% if remote_realm.realm_locally_deleted %}
+        <h4>Remote realm is locally deleted 🛑</h4>
+    {% endif %}
```

Comment:
I think we probably want similar checks for the other two fields here.
```
    # Whether the realm registration has been deactivated.                                              
    registration_deactivated = models.BooleanField(default=False)                                       
    # Whether the realm has been deactivated on the remote server.                                      
    realm_deactivated = models.BooleanField(default=False)                                              
    # Whether we believe the remote server deleted this realm                                           
    # from the database.                                                                                
    realm_locally_deleted = models.BooleanField(default=False)                                          
```

Reply:
Do we want a shared alert if any of these are `True` or is a separate line for each okay?

Reply:
I think a separate line for each is simple and fine.

Reply:
Cool. Pushed an update with a line for each field. I didn't manually test to see how it would look if all three were `True`.